apps/crn-server/src/routes/working-groups.route.ts

```diff
@@ -1,4 +1,5 @@
 import { validateFetchOptions } from '@asap-hub/server-common';
+import Boom from '@hapi/boom';
```

Comment:
what do you need this for?

Reply:
just leftover from when i had the endpoint here, will remove